### PR TITLE
action: add simple e2e ci

### PR DESCRIPTION
**Pull request by bergwolf**

### CODE REVIEW

**Comment by bergwolf** [comment]
https://github.com/bergwolf/nydus-snapshotter/runs/5418544506?check_suite_focus=true

File: misc/example/10-containerd-net.conflist [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "cniVersion": "0.3.1",
```
Comment (mikebrow) [inline misc/example/10-containerd-net.conflist:2]: 1.0.0

Comment (bergwolf) [reply misc/example/10-containerd-net.conflist:2]: Good point, thanks!

File: misc/example/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+## Example to setup and test nydus-snapshoter
```
Comment (imeoer) [inline misc/example/README.md:1]: Linking this doc on the main page should be better.

**Review by imeoer — APPROVED** [review]
LGTM!

**Pull request merged**